python/GafferUI/Widget.py

```diff
@@ -850,6 +852,38 @@ def _setStyleSheet( self ):
 
 		self.__qtWidget.setStyleSheet( _styleSheet )
 
+	@classmethod
+	def _getStyleClassName( cls ) :
```

Comment:
The presence of `get<Foo>` in Gaffer implies an equivalent `set<Foo>`. We can just call this `_styleClassName`. I'm presuming this is protected rather than private because of the `getattr()` below? Can we make it private somehow? Maybe use the mangled name in `getattr()`?

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/3229/commits/4f20764ab7a3c3fee66699cada79adc688d3198b